storage/innobase/xtrabackup/doc/source/glossary.rst

```diff
@@ -132,7 +132,7 @@
 
    .exp
```

Comment:
exp files are no longer generated and can be removed with all references (they were PS extension in 5.5). cfg files were replaced with .cfg files used in MySQL since 5.6